JobSeek/JobSeek.Api/Services/Implementation/EmployeeService.cs

```diff
@@ -0,0 +1,69 @@
+﻿using JobSeek.Api.Models.Entities;
+using JobSeek.Api.Models.Input;
+using JobSeek.Api.Repository.Contracts;
+using Mapster;
+
+namespace JobSeek.Api.Services.Implementation
+{
+    public class EmployeeService : BaseService<Employee, EmployeeInput>
+    {
+        private readonly IBaseRepository<Employee> _repository;
+
+        public EmployeeService(IBaseRepository<Employee> repository) : base(repository)
+        {
+            _repository = repository;
+        }
+
+        public Employee Create(EmployeeInput input)
+        {
+            var employee = input.Adapt<Employee>();
+
+            var employeeId = _repository.GetById(employee.Id);
+
+            if (employeeId == null)
+                throw new Exception("There is no Employee");
+
+            if (employee.Password.Length >= 8)
+                throw new Exception("Password must be more than 8 digits");
+
+            if (employee.Email.Contains('@') != false)
+                throw new Exception("Your email format is incorrect");
+
+            var dateNow = DateTimeOffset.Now;
+            if (employee.ModificationDate < dateNow)
```

Comment:
in input we don't have modificationDate. so user have not any access to it. and if you check the data context you will see that it get value automatically. so you dont need to check it